gm4_enchantment_extractors/data/enchantment_extractors/functions/init.mcfunction

```diff
@@ -0,0 +1,18 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing Enchantment Extractors..."}]
+execute unless entity @p run say GM4: Installing Enchantment Extractors...
+
+#declare and initialise scoreboards and settings
+scoreboard players set updateHappened gm4_upCheck 1
+scoreboard players set enchantment_extractors gm4_modules 1
+scoreboard players set enchantment_extractors gm4_clockTick 0
+
+scoreboard objectives add gm4_random dummy
```

Comment:
Like with desire lines, we can't use a generic scoreboard name here. How does gm4_ench_random sound?